### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.3][IntelliJ][Wipro]: Table commands are not working for tables present in select subscriptions window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1487]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to Tools menu, point to "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Interactive" radio button, and navigate to "Sign In" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. In the Select Subscriptions dialog , press tab to navigate tables then try to access the table cells by using table commands(Ctrl+Alt+Arrow keys) then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:**
By using below table command keys user should be able to access the table and NVDA should announced the Value present in the table cell then row position header name and column position
 Ctrl + Alt + Right Arrow-> Cell to right
Ctrl + Alt +Left Arrow->Cell to left
Ctrl + Alt +Down Arrow-> Cell below
Ctrl + Alt +Up Arrow->Cell above

**Actual Result:**
**NVDA Behavior:**
NVDA is not announcing anything by using table command keys and table keys are not working for the subscription table
**Jaws Behavior:** Jaws is announcing as expected
Subscription name
Not Selected DevDiv
 Key Vault
Column 2
Not Selected Java Dev Experience Tests with TTL = 7 Days
Row 2
Not Selected true
Column 1

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![select subscription-table](https://user-images.githubusercontent.com/22956130/36134921-c4303966-10ad-11e8-9a01-fb51737d10e7.jpg)
![table keys are not working in select subscriptions table](https://user-images.githubusercontent.com/22956130/36134967-0416b014-10ae-11e8-9abf-e8b178127e60.jpg)

**Comment by Eskibear** [comment]
duplicated with #1489

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749957
